docs/metrics/accelerate.md

```diff
@@ -9,80 +9,86 @@ Pulse calculates the four key Accelerate metrics using strictly **changes**, **d
 
 ## Deployment frequency
 
-How often your organization completes a deployment to production or releases code to end-users of your primary application or service.
+How often your organization completes a deployment to production or releases code to end-users of your primary application or service:
 
-The most specific that applies:
+```
+number of deployments per day
+```
```

Comment:
I'm using these code blocks as a convention for clearly identifying the "formulas" that we're using for calculating each metric.